test/service_manager_test.dart

```diff
@@ -166,7 +166,9 @@ void main() {
       await serviceManager.performHotReload();
     });
 
-    // TODO(kenzie): add hot restart test case.
+    test('hotRestart', () async {
+      await serviceManager.performHotRestart();
```

Comment:
verify that something happened. For example: 
if you use Eval to set a top level field in flutter, after you call hot restart it won't be set.

Reply:
Done.